config/webpack.config.js

```diff
@@ -189,6 +275,7 @@ const configGenerator = (buildOptions, apps) => {
       }),
       new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
     ],
+    devServer: generateWebpackDevConfig(buildOptions),
```

Comment:
Previously, we'd combine the two configs in the Metalsmith plugin. This is more semantic.